src/components/reference-book/page.cjsx

```diff
@@ -13,7 +13,7 @@ GetPositionMixin = require '../get-position-mixin'
 {ReferenceBookStore} = require '../../flux/reference-book'
 {ReferenceBookExerciseStore} = require '../../flux/reference-book-exercise'
 
-EXERCISE_MATCHER = new RegExp('#ost\/api\/ex\/(.*)')
+EXERCISE_MATCHER = new RegExp('\/exercises\/(.*)')
```

Comment:
I think it should because BE is doing something similar for book import. There might be a better CSS selector to pull it out. @karenc what's the XPath used? (feel free to add Dantemss if he's not busy with book reimport)